src/Controller/Speaker/Create.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Controller\Speaker;
+
+use App\Entity\Speaker;
+use App\Form\SpeakerType;
+use App\Repository\SpeakerRepository;
+use App\Service\FileUploader;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use Twig_Environment;
+
+final class Create
+{
+    private $renderer;
+    private $speakerRepository;
+    private $formFactory;
+    private $router;
+    private $fileUploader;
+
+    public function __construct(
+        Twig_Environment $renderer,
+        SpeakerRepository $speakerRepository,
+        FormFactoryInterface $formFactory,
+        RouterInterface $router,
+        FileUploader $fileUploader
+    )
```

Comment:
The closing bracket and opening curly brace should be on the same line in this case